packages/client/src/app/components/rating-icon/rating-icon.component.ts

```diff
@@ -9,6 +9,7 @@ import { ContentRating } from '@pulp-fiction/models/content';
 })
 export class RatingIconComponent implements OnInit {
     @Input() rating: ContentRating;
+    @Input() size: string;
```

Comment:
Super minor nitpick: this could be an enum like RatingIconSize or something (with appropriate string representations)